packages/core/src/connectors/index.ts

```diff
@@ -92,23 +92,39 @@ const loadConnectors = async () => {
 
 export const getLogtoConnectors = async (): Promise<LogtoConnector[]> => {
   const connectors = await findAllConnectors();
-  const connectorMap = new Map(connectors.map((connector) => [connector.id, connector]));
 
-  const logtoConnectors = await loadConnectors();
+  const loadConnectorPackages = await loadConnectors();
```

Comment:
what about the name `standardConnectors`? Indicating that they are not "named"

Reply:
And also this name can be used in API for listing all the available connectors to be added.